internal/platform/repositories/alert/telegram.go

```diff
@@ -28,8 +28,17 @@ func (r *Repository) CreateTelegramMessage(alert models.Alert) error {
 			params := url.Values{}
 			params.Add("chat_id", t.Chat)
 			params.Add("parse_mode", "markdownv2")
-			if t.Topic != "" {
-				params.Add("message_thread_id", t.Topic)
+			// Don't send message if the the topic is matched to any dropped rules.
+			if len(r.config.Notifier.Telegram.Drop) > 0 {
```

Comment:
According to previous comments, we need rules for each team. Also, rename the variable.